docs/development/codeguide.rst

```diff
@@ -212,30 +212,29 @@ appropriate.  Therefore, there is a global configuration option,
 to `False` by default.
 
 The following conventions should be used for classes that define the
-standard string conversion methods (``__str__``, ``__repr__``,
-``__bytes__``, and ``__format__``).  In the bullets
-below, the phrase "string instance" is used to refer to `str`, while
-"bytes instance" is used to refer to `bytes`.
+standard conversion methods:
 
-- ``__repr__``: Return a "string instance" containing only 7-bit characters.
+- `~object.__repr__`: Return a `str` instance containing only ASCII characters.
 
-- ``__bytes__``: Return a "bytes instance" containing only 7-bit characters.
+- `~object.__bytes__`: Return a `bytes` instance containing only ASCII
+  characters.
```

Comment:
I'm not sure if this guideline makes any sense in Python 3. If I am calling `bytes()` then I am not interested in string conversion, so why should the output of `bytes()` be limited to ASCII?

Reply:
The original author no longer in project, let me summon @dhomeier . 🙏